templates/website/index-index.html

```diff
@@ -26,6 +25,12 @@
 </div>
 <?php endif; ?>
 
+<div class="row row-full-width index-input" style="padding-bottom: 0;">
+    <div class="large-8 large-centered columns">
+        <h2 style="text-align: center; margin: 0;">Write to your politicians, national or local, for free.</h2>
```

Comment:
This should be the blurb-top commented out above, because it's not always the message given here. e.g. on https://www.writetothem.com/?a=council it is "Write to your Councillors"